activestorage/lib/active_storage/validations/attachment_content_type_validator.rb

```diff
@@ -0,0 +1,91 @@
+module ActiveStorage
+  module Validations
+    class AttachmentContentTypeValidator < BaseValidator
```

Comment:
Attachment Validators share a lot of common code, so I abstracted it to a common `BaseValidator`.